app/apollo/resolvers/channel.js

```diff
@@ -264,9 +307,11 @@ const channelResolvers = {
       const user = whoIs(me);
 
       try {
-        logger.info({ req_id, user, org_id, name }, `${queryName} validating`);
+        logger.info({req_id, user, org_id, name}, `${queryName} validating`);
+
+        await validAuth(me, org_id, ACTIONS.CREATE, TYPES.CHANNEL, queryName, context, [UUID(), name]);
```

Comment:
Why `UUID()` as a resource value?

Reply:
There isn't a defined uuid at this point. It is created right after checking auth. We want to check if user is allowed to create this before creating the resource.